### DISCUSSION TITLE
is this app effected by the autospill vulnerability?

### DISCUSSION THREAD

**Discussion by wucke13** [zeapo/Android-Password-Store#2813] [Q&A] [answered]
Here is some random article with an introduction: https://www.bleepingcomputer.com/news/security/autospill-attack-steals-credentials-from-android-password-managers/ . I haven't found a related CVE number yet.

**Comment by msfjarvis** [comment, marked as answer]
From what I can read I don't believe we're affected, but the authors of the exploit just straight up ignored Bitwarden so I don't have any other OSS project to cross reference my assessment against.
